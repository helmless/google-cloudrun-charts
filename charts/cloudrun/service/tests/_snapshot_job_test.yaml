suite: Cloud Run Job > Snapshot Tests
templates:
  - cloudrun.yaml
set:
  type: job
tests:
  - it: should render default manifest without errors
    asserts:
      - matchSnapshot: {}
