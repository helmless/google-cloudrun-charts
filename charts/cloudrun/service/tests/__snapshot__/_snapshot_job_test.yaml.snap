should render default manifest without errors:
  1: |
    apiVersion: run.googleapis.com/v1
    kind: Job
